backend/app/management/commands/runanalysis.py

```diff
@@ -43,85 +49,51 @@ def handle(self, *args, **options):
             print_header('There is no analysis with that name.')
             sys.exit(1)
 
-        if analysis_module:
```

Comment:
We used to save pickles of the analyses, so that we could restart analysis runs that were interrupted by a crash, but we haven't been using that feature, so I've removed it. We now do not overwrite existing analyses by default, but have a `--overwrite` flag to do so.